README.md

```diff
@@ -2,12 +2,15 @@
 
 Welcome to the SpatialOS and Unreal GDK community.
 
-Our SpatialOS community shapes the functionality of the Unreal GDK, so make sure you are part of the conversation. Come and find us on [Discord](https://discordapp.com/channels/311273633307951114/339471548647866368) or talk to us on the **Feedback** section in our [forums](https://forums.improbable.io/) (use the **unreal-gdk** tag - [this link](https://forums.improbable.io/new-topic?category=Feedback&tags=unreal-gdk) takes you there and pre-fills the category and tag).
+Our SpatialOS community shapes the functionality of the Unreal GDK, so make sure you are part of the conversation. Come and find us on [Discord](https://discordapp.com/channels/311273633307951114/339471548647866368) or talk to us on the **Feedback** section in our [forums](https://forums.improbable.io/) (use the **unreal-gdk** tag - [this link](https://forums.improbable.io/new-topic?category=Feedback&tags=unreal-gdk) takes you there and pre-fills the category and tag). You can also [sign up for SpatialOS Unreal GDK updates](link) direct to your mailbox.
 
 #### What's the Unreal GDK all about?
-The **SpatialOS Unreal GDK** is an Unreal Engine 4 (UE4) module made by Improbable. You can use the Unreal GDK to integrate persistent multiplayer worlds into your Unreal game using the SpatialOS platform. 
+The **SpatialOS Unreal GDK** is a module for Unreal Engine 4 (UE4) made by Improbable. You can use the Unreal GDK to integrate persistent multiplayer worlds into your Unreal game using the SpatialOS platform. 
+
+For more information on the Unreal GDK and its pre-alpha release, see our [blogpost](https://improbable.io/games/blog/unreal-gdk-pre-alpha) on the Improbable website.
+
+
 
-You can [sign up for SpatialOS Unreal GDK updates](link) direct to your mailbox.
```

Comment:
I just got the link for this are we not having it anymore?

Reply:
@danielimprobable I think your changes supersede @ElleEss's changes

Reply:
Updated